src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -377,25 +372,25 @@ export default class StandardRouteWaypointModel extends BaseModel {
      * - "S210"
      *
      * @for StandardRouteWaypointModel
-     * @method _parseWaypointRestrictions
+     * @method _setRestrictions
      * @param waypointRestrictions {string}
      * @private
      */
-    _parseWaypointRestrictions(waypointRestrictions) {
+    _setRestrictions(waypointRestrictions) {
```

Comment:
We are sorting between the two types of restrictions, and calling `_setAltitudeRestriction()` and `_setSpeedRestriction()` as applicable. Personally, I would say it makes sense to call the caller `setRestrictions()` because in plain speak, that really is what it's doing. It's actually from the same basis of logic that I changed it: we're not really parsing anything, just accepting data and setting the property values with that data. Perhaps neither is a good name? What about something like `applyRestrictions()` or something?